tyk-pro/values.yaml

```diff
@@ -16,7 +16,10 @@ fullnameOverride: ""
 
 # Master switch for enabling/disabling bootstraing job batch, role binding,
 # role, and account service.
-bootstrap: true
+bootstrap:
+  jobName: bootstrap-post-install
```

Comment:
Why are we making jobName configurable?

Reply:
It is needed to be passed to the deletion job in order to know what jobs to delete and I believed it is safer to have a single place to modify that having to keep track of all the places where we should modify it.

Reply:
Won't it be backward incompatible change?

Reply:
I agree with @komalsukhani this will break previous charts.

Reply:
I have adapted the file for backwards compatibility.